tpm2/test/tpm2_test.go

```diff
@@ -2111,3 +2111,87 @@ func TestPolicyOr(t *testing.T) {
 		t.Errorf("PolicyOr with error: %v", err)
 	}
 }
+
+func TestNVUndefineSpaceSpecial(t *testing.T) {
+	rw := openTPM(t)
+	defer rw.Close()
+
+	// Set up policy session for creation (Trail session)
+	sess, _, err := StartAuthSession(rw, HandleNull, HandleNull, make([]byte, 16), nil, SessionTrial, AlgNull, AlgSHA256)
+	if err != nil {
+		t.Errorf("StartAuthSession() for policy generation failed: %v", err)
+	}
+	// Use well known SHA256 values to generate a known policy.
+	hashVal := make([]byte, 32)
+	for iterator := range hashVal {
+		hashVal[iterator] = byte(iterator)
+	}
+	hashNull := make([]byte, 32)
+	// Use well known hash for policy generation with PolicyOr
+	err = PolicyOr(rw, sess, TPMLDigest{Digests: []tpmutil.U16Bytes{tpmutil.U16Bytes(hashVal), tpmutil.U16Bytes(hashNull)}})
```

Comment:
If I understand the spec correctly PolicyOR hashes all the digests and resets the policyHash of the session to ZeroDigest.
See TPM2 Spec, Part3 - Commnds, Page 227. Especially `Note 2`

It looks like because the policyDigest is set so zeroDigest I have full controll about the policyDigest of the session.